docs/products/kafka/kafka-connect/reference/s3-sink-formats.rst

```diff
@@ -0,0 +1,9 @@
+AWS S3 sink connector naming and data formats
+=============================================
+
+The Apache Kafka Connect® S3 sink connector enables you to move data from an Aiven for Apache Kafka® cluster to Amazon S3 for long term storage. 
+
+Aiven provides, as managed service, two version of the S3 sink connector, each one having different naming and data formats:
```

Comment:
Same as above, position of "as managed service" sounds a bit off to me, either move it to the front, or remove (seems not so much relevant to me)

Reply:
fixed